packages/devel/gperftools/package.mk

```
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2021-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="gperftools"
+PKG_VERSION="gperftools-2.9.1"
+PKG_SHA256="484a88279d2fa5753d7e9dea5f86954b64975f20e796a6ffaf2f3426a674a06a"
+PKG_LICENSE="BSD"
+PKG_SITE="https://github.com/gperftools/gperftools"
+PKG_URL="https://github.com/gperftools/gperftools/archive/${PKG_VERSION}.tar.gz"
+PKG_DEPENDS_TARGET="toolchain"
+PKG_LONGDESC="Google Performance Tools"
+PKG_TOOLCHAIN="autotools"
+
+PKG_CONFIGURE_OPTS_TARGET="--enable-minimal --disable-debugalloc --disable-static"
```

```suggestion
PKG_CMAKE_OPTS_TARGET="-DBUILD_DEBUGALLOC=OFF"
```

it builds :) no idea if it works